specifyweb/frontend/js_src/lib/components/QueryBuilder/helpers.ts

```diff
@@ -68,7 +69,8 @@ export function parseQueryFields(
          */
         const startValue =
           typeof field.startValue === 'string' &&
-          fieldSpec.datePart === 'fullDate'
+          fieldSpec.datePart === 'fullDate' &&
+          !field.startValue.includes(today)
```

Comment:
could you do `startsWith` instead of `includes`?

1. the behavior is the same for valid value
2. does not mistrigger it invalid values
3. the code also serves as documentation (i.e, says that `today` can only appear at the beginning, not anywhere in the string)